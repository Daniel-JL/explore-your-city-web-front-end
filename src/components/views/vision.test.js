import React from 'react';
import renderer from 'react-test-renderer';
import {
  MemoryRouter,
} from 'react-router-dom';
import Vision from './vision';

describe('Contribution page', () => {
  it('should render correctly', () => {
    const wrapper = renderer.create(
      <MemoryRouter>
        <Vision />
      </MemoryRouter>,
    ).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
