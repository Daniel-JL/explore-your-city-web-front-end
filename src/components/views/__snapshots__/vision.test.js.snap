// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contribution page should render correctly 1`] = `
<div
  className="sc-fubCfw jkgfuu"
>
  <div
    className="sc-pFZIQ fzjIby"
  >
    <div
      className="sc-jrAGrp kuRXtY"
    >
      <img
        className="sc-kEjbxe bdzfem"
      />
    </div>
    <div
      className="sc-iqHYGH bdPUrq"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam congue tortor eu consequat faucibus. Cras scelerisque elit magna, a aliquet massa consectetur et. Sed in nisl a lorem fermentum ullamcorper in et diam. Phasellus vitae vulputate felis. Suspendisse efficitur, ligula at semper ornare, elit nisl euismod enim, eget maximus augue felis non quam. Donec dictum nibh vitae magna scelerisque, ac aliquam ipsum consectetur. Donec et tellus nec tellus scelerisque posuere sagittis vel nisi. Proin quis pretium urna, eu laoreet nibh. Maecenas nec vestibulum lectus. Fusce aliquet nec erat ut ullamcorper. Vestibulum consectetur tincidunt dapibus.
    </div>
  </div>
  <div
    className="sc-pFZIQ fzjIby"
  >
    <div
      className="sc-iqHYGH bdPUrq"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam congue tortor eu consequat faucibus. Cras scelerisque elit magna, a aliquet massa consectetur et. Sed in nisl a lorem fermentum ullamcorper in et diam. Phasellus vitae vulputate felis. Suspendisse efficitur, ligula at semper ornare, elit nisl euismod enim, eget maximus augue felis non quam. Donec dictum nibh vitae magna scelerisque, ac aliquam ipsum consectetur. Donec et tellus nec tellus scelerisque posuere sagittis vel nisi. Proin quis pretium urna, eu laoreet nibh. Maecenas nec vestibulum lectus. Fusce aliquet nec erat ut ullamcorper. Vestibulum consectetur tincidunt dapibus.
    </div>
    <div
      className="sc-jrAGrp kuRXtY"
    >
      <img
        className="sc-kEjbxe bdzfem"
      />
    </div>
  </div>
  <a
    aria-current={null}
    href="/register"
    onClick={[Function]}
  >
    <button
      className="sc-bdfBwQ sc-iBPRYJ dDwjPX RzFWu"
    >
      Tell your own Story
    </button>
  </a>
</div>
`;
